firebase_admin/_messaging_utils.py

```diff
@@ -179,6 +164,36 @@ def __init__(self, title=None, body=None, icon=None, color=None, sound=None, tag
         self.title_loc_args = title_loc_args
         self.channel_id = channel_id
         self.image = image
+        self.ticker = ticker
+        self.sticky = sticky
+        self.event_timestamp = event_timestamp
+        self.local_only = local_only
+        self.priority = priority
+        self.vibrate_timings_millis = vibrate_timings_millis
+        self.default_vibrate_timings = default_vibrate_timings
+        self.default_sound = default_sound
+        self.light_settings = light_settings
+        self.default_light_settings = default_light_settings
+        self.visibility = visibility
+        self.notification_count = notification_count
+
+
+class LightSettings(object):
+    """Represents settings to control notification LED that can be included in a
+    ``messaging.AndroidNotification``.
+
+    Args:
+        color: Set color of the LED in ``#rrggbb`` or ``#rrggbbaa`` format (required).
```

Comment:
Drop the `(required)` bits. We usually only indicate things that are `(optional)`.